### PR TITLE
VIH-5946 new vho command centre

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-5983
### Change description ###
 - moved copy uid to separate component.
 - updated the ui for hearings list
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    width: 17px;
+    height: 20px;
+}
+.copy-box {
+    display: none;
+    top: 557px;
+    left: 139px;
+  }
+.copyIDbox {
+    font-family: "GDS Transport", Arial, sans-serif;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss:13]: Use the mixins instead of setting each style
`@include govuk-font(14, $weight: 'regular', $tabular: false, $line-height: false);`

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing-summary.spec.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ describe('HearingSummary', () => {
         expect(hearing.scheduledStartTime).toEqual(c.scheduled_date_time);
         expect(hearing.scheduledEndTime).toBeDefined();
         expect(hearing.hearingVenueName).toBe(c.hearing_venue_name);
+        expect(hearing.startedDateTime).toEqual(c.started_date_time);
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing-summary.spec.ts:22]: it might be worth adding a test to verify the model returns a null if the start and end time is not set rather than risk errors being thrown

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-network-status/participant-network-status.component.scss [hunk 3]
```diff
@@ -46,3 +46,34 @@
   height: auto;
   // margin-top: -1.8vw;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-network-status/participant-network-status.component.scss:47]: remove this commented style if we don't need it

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.scss [hunk 4]
```diff
@@ -134,3 +134,186 @@
...
+  -webkit-font-smoothing: antialiased;
+
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.11111;
+  margin-top: 0;
+}
+.listItemGreySmall {
+  color: govuk-colour("dark-grey");
+  font-family: "GDS Transport", Arial, sans-serif;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.scss:178]: same as before, use mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss [hunk 5]
```diff
@@ -10,11 +10,7 @@
     left: 139px;
   }
 .copyIDbox {
-    font-family: "GDS Transport", Arial, sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-weight: 400;
-    font-size: 14px;
+    @include govuk-font(14, $weight: 'bold', $tabular: false, $line-height: false);
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss:13]: should this be bold? I thought this was regular weighting?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html [hunk 6]
```diff
@@ -4,10 +4,10 @@
 
     <div class="shareIcon"><app-copy-id [conference]="conference"></app-copy-id></div> 
     <div class="time listItem" id="{{ conference.id }}-time">{{ conference.scheduledDateTime | date: 'HH:mm' }}</div>
-    <div class="started listItemGreySmall" id="{{ conference.id }}-started">started</div>
-    <div class="startedTime listItemGreySmall" id="{{ conference.id }}-startedTime">{{ conference.startedDateTime | date: 'HH:mm' }}</div>
-    <div class="ended listItemGreySmall" id="{{ conference.id }}-ended">ended</div>
-    <div class="endedTime listItemGreySmall" id="{{ conference.id }}-endTime">{{ conference.endedDateTime | date: 'HH:mm' }}</div>
+    <div *ngIf="conference.startedDateTime" class="started listItemGreySmall" id="{{ conference.id }}-started">started</div>
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html:7]: It would be better to have a single ngIf for startedDateTime and endedDateTime rather than twice.

Have you looked at ng-container

Something like

<ng-container *ngIf="conference.startedDateTime">
    <div>started text</div>
    <div>started time</div>
</ng-container>

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss (hunk 1 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss]: updated to use mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-network-status/participant-network-status.component.scss (hunk 3 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-network-status/participant-network-status.component.scss:47]: removed the style

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss (hunk 5 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss:13]: updated to regular,

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html (hunk 6 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html:7]: updated to ng-container

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.scss (hunk 4 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.scss:178]: replaced with mixins.
